sql/group_test.go

```diff
@@ -58,3 +61,128 @@ func TestDesiredAggregateOrder(t *testing.T) {
 		}
 	}
 }
+
+const testDatumCount = 1000
```

Comment:
other benchmarks use the style of having a count appended to their name, with the helpers taking the count as an argument. something like
```
func BenchmarkAvgAggregateInt1K(b *testing.B) {
  runBenchmarkAggregate(b, newAvgAggregate, makeIntTestData(1000))
}
```